Sming/SmingCore/Platform/System.cpp

```diff
@@ -51,7 +51,7 @@ void SystemClass::setCpuFrequency(CpuFrequency freq)
 	else
 		REG_CLR_BIT(0x3ff00014, BIT(0));
 
-	os_update_cpu_frequency(freq);
+	ets_update_cpu_frequency(freq);
```

Comment:
It seems that this function, along with others, was removed from osapi.h. An abbreviated version of the diff is given below. Lines starting with "-" mean that the code was removed and lines starting with "+" mean that this code was added. I guess we have to ask Espressif for the reasons behind these changes.

```diff
diff -u /opt/Espressif/esp_iot_sdk_v1.0.1/include/osapi.h /opt/Espressif/esp_iot_sdk_v1.1.0/include/osapi.h
--- /opt/Espressif/esp_iot_sdk_v1.0.1/include/osapi.h	2015-04-21 14:45:08.000000000 +0000
+++ /opt/Espressif/esp_iot_sdk_v1.1.0/include/osapi.h	2015-05-22 19:45:30.000000000 +0000
-#define os_install_putc2 ets_install_putc2
-#define os_intr_lock ets_intr_lock
-#define os_intr_unlock ets_intr_unlock
-#define os_isr_attach ets_isr_attach
-#define os_isr_mask ets_isr_mask
-#define os_isr_unmask ets_isr_unmask

-#define os_putc ets_putc
-#define os_str2macaddr ets_str2macaddr
-#define os_timer_done ets_timer_done
-#define os_timer_handler_isr ets_timer_handler_isr
-#define os_timer_init ets_timer_init
-#define os_update_cpu_frequency ets_update_cpu_frequency
+unsigned long os_random(void);
+int os_get_random(unsigned char *buf, size_t len);
```